packages/mgt-element/src/components/baseComponent.ts

```diff
@@ -249,4 +280,40 @@ export abstract class MgtBaseComponent extends LitElement {
     this._isLoadingState = value;
     this.requestUpdate('isLoadingState');
   }
+
+  /**
+   *  Gathers default strings from component for localization
+   *
+   * @protected
+   * @param {*} strings
+   * @returns
+   * @memberof MgtBaseComponent
+   */
+  protected serveStrings(strings) {
+    return LocalizationHelper.updateStringsForTag(this.tagName, strings);
+  }
+
+  /**
+   * returns directional attribute from Localization Helper
+   *
+   * @protected
+   * @returns
+   * @memberof MgtBaseComponent
+   */
+  protected getDirection() {
+    return LocalizationHelper.getDirection();
+  }
+
+  /**
+   * Adds rtl class to component if direction is returned rtl
+   *
+   * @private
+   * @memberof MgtBaseComponent
+   */
+  protected updateDirection() {
+    const direction = this.getDirection();
+    if (direction == 'rtl') {
+      this.classList.add('rtl');
```

Comment:
I notice there is no case to remove the RTL class. Is that intentional?

Reply:
I didn't think it'd be necessary along the same line of logic that we decided not to use the mutationObserver. We don't/shouldn't expect the user to be dynamically changing the `dir` value. 

Open to changing though. Any thoughts @nmetulev ?

Reply:
Updated LocalizationHelper here:

3154d42

firing only on dir mutation:
```
    if (this._isDirectionInit) {
      return;
    }

    this._isDirectionInit = true;
    this.mutationObserver = new MutationObserver(function(mutations) {
      mutations.forEach(function(mutation) {
        if (mutation.attributeName == 'dir') {
          this._directionEventDispatcher.fire(null);
        }
      });
    });
    this.mutationObserver.observe(document.body, { childList: true });
```